### ISSUE TITLE
Regression: Relative paths are not resolved in 2.1.3

### ISSUE THREAD

**Issue by apptaro** [KhronosGroup/COLLADA2GLTF#200]
I have two files:
```
D:\pathTo\model.dae
D:\pathTo\texture1.jpg
```
In model.dae, we have:
```
<library_images>
  <image id='texture1-image' name='texture1'>
    <init_from>./texture1.jpg</init_from>
  </image>
</library_images>
```
and COLLADA2GLTF-v2.1.3-windows-Release-x64 shows the error like this:
```
D:\dir>COLLADA2GLTF-bin.exe -i "D:\pahtTo\model.dae" -o "D:\pathTo\model.gltf" -v 2.0
Converting D:\pathTo\model.dae -> D:\pathTo\model.gltf
WARNING: Image uri: .\texture1.jpg could not be resolved
Time: 10 ms

D:\dir>
```
COLLADA2GLTF-v2.1.2-windows-Release-x64 worked without this issue:
```
D:\dir>COLLADA2GLTF-bin.exe -i "D:\pahtTo\model.dae" -o "D:\pathTo\model.gltf" -v 2.0
Converting D:\pathTo\model.dae -> D:\pathTo\model.gltf
Time: 5 ms

D:\dir>
```

**Comment by lasalvavida** [comment]
This will be fixed, one note though: this only occurs if you specify the input path as an absolute path.

So for example:

```bash
COLLADA2GLTF-bin.exe pathTo/model.dae
```

will convert correctly.

**Comment by lasalvavida** [comment]
This should be fixed in #202

**Comment by lasalvavida** [comment]
Resolved in 2.1.4.
